src/www/ui/api/index.php

```diff
@@ -168,6 +168,7 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
   function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->get('', GroupController::class . ':getGroups');
     $app->post('', GroupController::class . ':createGroup');
+    $app->delete('/{id:\\d+}', GroupController::class . ':deleteGroup');
```

Comment:
Please add a catch all request to handle invalid parameters. Currently, `DELETE /api/v1/groups/13a` returns a 500 error instead of a 400.

```suggestion
    $app->delete('/{id:\\d+}', GroupController::class . ':deleteGroup');
    $app->any('/{params:.*}', BadRequestController::class);
```

Reply:
A reminder @dushimsam , please add the function call to `any()`.

Reply:
done

Reply:
done